Common/Securities/Security.cs

```diff
@@ -618,6 +632,32 @@ public void SetDataNormalizationMode(DataNormalizationMode mode)
             {
                 subscription.DataNormalizationMode = mode;
             }
+
+            if (mode == DataNormalizationMode.Adjusted) return;
+
+            if (PriceVariationModel is AdjustedPriceVariationModel)
+            {
+                SetLiveModePriceVariationModel(true);
+            }
+        }
+
+        /// <summary>
+        /// In LiveMode, we set up the security price variation model to assure that 
+        /// order prices are rounded to meet brokerages precision requirements
+        /// </summary>
+        /// <param name="liveMode">True if live mode</param>
+        public void SetLiveModePriceVariationModel(bool liveMode)
```

Comment:
Please remove this and when the security is constructed setup the PriceVariationModel. Follow the patterns established by all the other model properties.